modules/special-rules/pages/war-engines.adoc

```diff
@@ -1,2 +1,3 @@
 = War engines
 
+{blank}[TODO: When we work on this topic, it may also be a good place to state that all War Engine units have the {no-retreat} and {unbreakable} abilities. It'd be a bit redundant to have a whole topic just to state that, but I think that we need some mention in place of the original E40k "No Retreat, No Surrender" topic.]
```

Comment:
For what it's worth, I'd like to add a special abilities section to all the war engine data sheets to list special abilities, and we could put them there instead of having a topic?

Reply:
Well, I didn't spell it out here but my underlying thought was that perhaps the best answer is 'both'? We definitely ought to add the abilities to the data sheets, I agree — but I think it might also be good to state very clearly in the rules that **all** War Engine units have both of these abilities. It is arguably slightly redundant but it is only one sentence and makes the 'global' principle very clear and in continuity with the original, so I'd argue that it is worth it?

Reply:
Both is fine, that was just a suggestion.

If you want to reduce the number of war engine topics, add it as a partial to War engines in close combat and War engines in firefights, as that's where both abilities take effect.

Reply:
>If you want to reduce the number of war engine topics, add it as a partial to War engines in close combat and War engines in firefights, as that's where both abilities take effect.

It's not so much to reduce the number of WE topics, more to not have a topic that has only a title and a single sentence in it if that info could fit quite neatly alongside some other general war engine info - hence `war-engines.adoc` seemed like a good fit. But I see what you mean -- if we put it in a partial for those two topics then it'd be right where it is most relevant. Good idea, that probably will work even better.

Reply:
I'll apply this suggestion to capture the point for later, and then resolve this conversation to tidy up:
```suggestion
{blank}[TODO: When we work on this topic, it may also be a good place to state that all War Engine units have the {no-retreat} and {unbreakable} abilities. It'd be a bit redundant to have a whole topic just to state that, but I think that we need some mention in place of the original E40k "No Retreat, No Surrender" topic. Another option as Ian pointed out is to add the statement to a partial and use it in both "War engines in close combat" and "War engines in firefights" topics, where the abilities both take effect.]
```